exercises/robot-name/RobotNameTest.cs

```diff
@@ -30,4 +31,22 @@ public void Can_reset_the_name()
         robot.Reset();
         Assert.NotEqual(originalName, robot.Name);
     }
+
+    [Fact(Skip = "Remove to run test")]
+    public void After_reset_the_name_is_valid()
+    {
+        robot.Reset();
+        Assert.Matches(@"^[A-Z]{2}\d{3}$", robot.Name);
+    }
+
+    [Fact(Skip = "Remove to run test")]
+    public void Robot_names_are_unique()
+    {
+        var names = new HashSet<string>();
+        for (int i = 0; i < 10_000; i++) {
+            var robot = new Robot();
+            Assert.DoesNotContain(names, name => name == robot.Name);
```

Comment:
```suggestion
            Assert.DoesNotContain(name, names);
```

Reversing the argument allows you to get rid of the lambda.